### DISCUSSION TITLE
spring boot 2.7.12使用apollo 2.1.0获取不到logging.config配置

### DISCUSSION THREAD

**Discussion by windy-zhh** [apolloconfig/apollo#4928] [Q&A]
在apollo上配置了logging.config=classpath:log4j2-dev.xml发现没有生效，springboot启动的时候还是使用了默认的log4j2.xml作为日志配置
项目的log4j2.xml里面的console配置：
`<Console name="console" target="SYSTEM_OUT">
            <PatternLayout pattern="%d %t [%F:%L] %C %-5p : [%X{traceId}] %m%n" />
</Console>`
项目的log4j2-dev.xml里面的console配置：
`<Console name="console" target="SYSTEM_OUT">
            <PatternLayout pattern="%d %t [%F:%L] %C %-5p : [%X{traceId},%X{spanId},%X{parentId},%X{exportable}] %m%n" />
</Console>`

测试启动时候的console日志如下，没有读到log4j2-dev.xml：
![image](https://github.com/apolloconfig/apollo/assets/6142624/f3e685db-b046-43f0-aa0d-b35b7808ab11)

加上apollo.bootstrap.eagerLoad.enabled=true后发现在打印springboot之前可以读取并使用log4j2-dev.xml，之后仍然使用log4j2.xml：
![image](https://github.com/apolloconfig/apollo/assets/6142624/5dc0e044-9bfc-42da-a063-970f87118e63)

springboot版本：2.7.12
apollo版本：2.1.0
spring cloud版本：2021.0.7

**Comment by nobodyiam** [comment]
可以参考下 https://github.com/apolloconfig/apollo-use-cases/tree/master/spring-boot-logger

**Reply by qiang999** [reply]
@nobodyiam 我现在也遇到了这个问题，也就是apollo目前不支持指定logging.config文件，只是支持把里面需要动态更新的内容放到apollo配置中心？
